person.rb

```diff
@@ -0,0 +1,32 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: false)
```

Comment:
- [ ] **Good job so far 👏 👏** However, it would be better to set the default value of `parent_permission: true` as suggested in the project requirements 😂